CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+# dbt_facebook_ads_source v0.8.0
+
+[PR #36](https://github.com/fivetran/dbt_facebook_ads_source/pull/36) includes the following updates:
+
+## Breaking Changes 
+- Incorporates the `basic_ad_actions` and `basic_ad_action_values` pre-built reports in order to grab conversion data. They are both child tables of the already-required `basic_ad` report, broken down by `action_type`. 
+  - Addition of the new `stg_facebook_ads__basic_ad_actions` and `stg_facebook_ads__basic_ad_action_values` (and their `_tmp` counterparts) staging models. Given that this is a schema change for the package, this a breaking change.
+
+## Feature Updates: Conversion Metrics
+- With the inclusion of the `basic_ad_actions` and `basic_ad_action_values` source tables, we have added the `facebook_ads__basic_ad_actions_passthrough_metrics` and `facebook_ads__basic_ad_action_values_passthrough_metrics` variables to pass through additional conversion value metrics to downstream models.
+  - By default, the package includes only the conversion value calculated using the default attribution window, but your report may include calculations using the other windows defined [here](https://developers.facebook.com/docs/marketing-api/reference/ads-action-stats/). See [README](https://github.com/fivetran/dbt_facebook_ads_source/tree/main?tab=readme-ov-file#passing-through-additional-metrics) for details on how to use the new variables.
```

Comment:
very minor but do we want to just mention what the default attribution window is

Reply:
adding a link to our connector docs about it